src/plugins/kubeconfig/index.ts

```diff
@@ -109,14 +111,12 @@ export const requestAccessToCluster = async (
   if (!response) {
     throw "Did not receive access ID from server";
   }
-  const { id, isPreexisting } = response;
-  if (!isPreexisting) {
-    print2(
-      "Waiting for access to be provisioned. This may take up to a minute."
-    );
-  }
```

Comment:
Why can the isPreexisting be removed?

Reply:
Isn't `isPreexisting` about approval, not provisioning? Seems like you want to print a "waiting for approval" message when requesting access only if `!preExisting`, but always print a "waiting for ... to be provisioned" whenever an access request is made.

Reply:
Yes, I believe it's about the approval, not provisioning per se, so the original message wasn't really right either. Either way, the real reason for my removing this is that it was intended as a way to tell the user that the CLI isn't frozen, just that it's waiting for the request to complete, but this is now taken care of by the `spinUntil()` call below this.